<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefTutorBase</class>
 <widget class="QWidget" name="PrefTutorBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>400</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="kcfg_TutorTiming">
     <property name="title">
      <string>Tutor Timing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButtonEvery">
        <property name="text">
         <string>&amp;Every</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="kcfg_TutorEvery"/>
        </item>
        <item>
         <widget class="QLabel" name="labelMinutesPeriodic">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;minutes</string>
          </property>
          <property name="buddy">
           <cstring>kcfg_TutorEvery</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="radioButtonRandomly">
        <property name="text">
         <string>&amp;Randomly</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelBetween">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Between</string>
          </property>
          <property name="setVisible()" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_TutorMin">
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMinutesMax">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>m&amp;inutes and</string>
          </property>
          <property name="buddy">
           <cstring>kcfg_TutorMin</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_TutorMax">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelMinutesMin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>mi&amp;nutes</string>
          </property>
          <property name="buddy">
           <cstring>kcfg_TutorMax</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="kcfg_TutorCardAppearance">
     <property name="title">
      <string>Flashcard Appearance</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonAppearanceNative">
        <property name="text">
         <string>&amp;Native</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButtonAppearanceMinimalistic">
        <property name="text">
         <string>Minim&amp;alistic</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBoxMisc">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="kcfg_StartExerciseAsSoonAsFileIsLoaded">
        <property name="text">
         <string>S&amp;tart exercise as soon as vocabulary file is opened</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>kcfg_StartExerciseAsSoonAsFileIsLoaded</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
